diff --git a/node_modules/chia-client/dist/src/FullNode.js b/node_modules/chia-client/dist/src/FullNode.js
index 547fc0c..17bae75 100644
--- a/node_modules/chia-client/dist/src/FullNode.js
+++ b/node_modules/chia-client/dist/src/FullNode.js
@@ -77,6 +77,9 @@ class FullNode extends RpcClient_1.RpcClient {
             header_hash: hash,
         });
     }
+    async getNetworkInfo() {
+        return this.request("get_network_info", {});
+    }
     /* https://github.com/CMEONE/chia-utils */
     addressToPuzzleHash(address) {
         return chia_utils_1.address_to_puzzle_hash(address);
diff --git a/node_modules/chia-client/dist/src/Wallet.js b/node_modules/chia-client/dist/src/Wallet.js
index 1eba50f..9cfbc23 100644
--- a/node_modules/chia-client/dist/src/Wallet.js
+++ b/node_modules/chia-client/dist/src/Wallet.js
@@ -100,6 +100,10 @@ class Wallet extends RpcClient_1.RpcClient {
         const { transactions } = await this.request("get_transactions", { wallet_id: walletId, end: limit });
         return transactions;
     }
+    async getAddress(walletId) {
+        const { address } = await this.request("get_next_address", { wallet_id: walletId, new_address: false });
+        return address;
+    }
     async getNextAddress(walletId) {
         const { address } = await this.request("get_next_address", { wallet_id: walletId, new_address: true });
         return address;
